apps/SageAccountWeb/src/components/ResetPassword.tsx

```diff
@@ -96,55 +63,12 @@ export const ResetPassword = (props: ResetPasswordProps) => {
           token ? (
             <>
               <SourceAppLogo />
-              <form onSubmit={handleChangePasswordWithToken}>
-                <StyledFormControl
-                  fullWidth
-                  required
-                  variant="standard"
-                  margin="normal"
-                  sx={formControlSx}
-                >
-                  <TextField
-                    fullWidth
-                    required
-                    type="password"
-                    id="newPassword"
-                    name="newPassword"
-                    label={'New password'}
-                    onChange={e => setNewPassword(e.target.value)}
-                    value={newPassword || ''}
-                  />
-                </StyledFormControl>
-                <StyledFormControl
-                  fullWidth
-                  required
-                  variant="standard"
-                  margin="normal"
-                  sx={formControlSx}
-                >
-                  <TextField
-                    fullWidth
-                    required
-                    type="password"
-                    id="confirmPassword"
-                    name="confirmPassword"
-                    label={'Confirm password'}
-                    onChange={e => setConfirmPassword(e.target.value)}
-                    value={confirmPassword || ''}
-                  />
-                </StyledFormControl>
-                <Button
-                  className="btn-container"
-                  variant="contained"
-                  type="submit"
-                  fullWidth
-                  onSubmit={handleChangePasswordWithToken}
-                  sx={buttonSx}
-                  disabled={!newPassword || !confirmPassword}
-                >
-                  Change Password
-                </Button>
-              </form>
+              <ChangePasswordWithToken
```

Comment:
Use SRC component to handle changing password. This component just intercepts the token from the URLSearchParams and passes it along.